modules/cratedb/src/test/java/org/testcontainers/junit/cratedb/SimpleCrateDBTest.java

```diff
@@ -34,7 +34,7 @@ public void testSimple() throws SQLException {
     public void testCommandOverride() throws SQLException {
         try (
             CrateDBContainer cratedb = new CrateDBContainer(CrateDBTestImages.CRATEDB_TEST_IMAGE)
-                .withCommand("crate -C cluster.name=testcontainers")
+                .withCommand("crate -C discovery.type=single-node cluster.name=testcontainers")
```

Comment:
applying the command in `CrateDBContainer` works :D but this test will fail because there is no cluster

Reply:
apologies, fixed!